src/main/java/uk/gov/companieshouse/charges/data/service/ChargesService.java

```diff
@@ -165,7 +165,7 @@ private void saveAndInvokeChsKafkaApi(String contextId, String companyNumber,
                 companyNumber,
                 chargeId);
         HttpStatus httpStatus = HttpStatus.resolve(res.getStatusCode());
-        if (httpStatus == null || !httpStatus.is2xxSuccessful()) {
+        if (httpStatus == null || !(httpStatus.is2xxSuccessful() || httpStatus.value() == 503)) {
```

Comment:
Not sure why we should handle 503 specially?

Reply:
Its not handling the 503 stopping it being a 500!

Reply:
Stops it being thrown as a ResponseStatusException